config/v1/types_scheduling.go

```diff
@@ -48,6 +48,14 @@ type SchedulerSpec struct {
 	// would not be applied.
 	// +optional
 	DefaultNodeSelector string `json:"defaultNodeSelector,omitempty"`
+	// MastersSchedulable allows masters nodes to be schedulable. This means, if there
+	// are n no of master nodes in the cluster, all of them would be made schedulable.
+	// so that workload pods can run on them. 
+	// Important Note: Once the workload pods start running on the master nodes, there
+	// is a very good chance that performance of control plane components is impacted.
+	// Please turn on this field after doing due deligence.
```

Comment:
run a spell check on your doc.  diligence isn't it?

Reply:
Yes it is. I changed it. Thanks for catching it.

Reply:
and what do you use for spell checking? I haven't tried anything yet of that sort and something for fedora?